ps_landing_page/models/hr_employee_landing_page.py

```diff
@@ -260,29 +306,42 @@ def action_view_analytic_tree(self):
         self.ensure_one()
         ir_model_data = self.env["ir.model.data"]
         tree_res = ir_model_data.get_object_reference(
-            "ps_timesheet_invoicing", "view_ps_time_line_tree"
+            "ps_landing_page", "view_ps_time_line_landing_page_tree"
         )
         tree_id = tree_res and tree_res[1] or False
 
         user_id = self.env.user.id
+        # self.env.cr.execute(
+        #     """
+        #     SELECT
+        #         id
+        #         FROM ps_time_line
+        #         WHERE user_id = %s
+        #           AND ot = true
+        #     UNION
+        #     SELECT
+        #         ps_time_line.id
+        #         FROM ps_time_line
+        #         JOIN project_project
+        #         ON ps_time_line.project_id=project_project.id
+        #         AND overtime
+        #         WHERE ps_time_line.user_id = %s
+        #         AND ps_time_line.state != 'draft'
+        #     """,
+        #     (user_id, user_id),
+        # )
```

Comment:
never commit commented code. This makes the file harder to read, and if you ever want to go back to previous versions, we have git.

Reply:
Noted!